src/runtime/key-bag.ts

```
@@ -10,33 +11,218 @@ import {
+    if (keyBagAction) {
+      this.keybag._upsertNamedKey(this);
+    }
+    return Result.Ok({
+      modified: keyBagAction && true,
+      kfp: found,
+    });
+  }
+
+  async asKeysItem(): Promise<KeysItem> {
+    const my = { ...this.keys };
+    delete my["*"];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)